deny.toml

```diff
@@ -75,6 +75,9 @@ skip-tree = [
     # wait for indexmap, toml_edit, serde_json, tower to upgrade
     { name = "hashbrown", version = "=0.12.3" },
 
+    # wait for jsonrpc-http-server to upgrade
+    { name = "redox_syscall", version = "=0.2.16" },
+
```

Comment:
This duplicate dependency exception is already present at line 70. Was this accidentally caused by a git merge?

Reply:
```suggestion
```